airflow/www/utils.py

```diff
@@ -76,6 +76,128 @@ def is_accessible(self):
         )
 
 
+def generate_pages(current_page, num_of_pages,
+                   search=None, showPaused=None, window=7):
+    """
+    Generates the HTML for a paging component using a similar logic to the paging
+    auto-generated by Flask managed views. The paging component defines a number of
+    pages visible in the pager (window) and once the user goes to a page beyond the
+    largest visible, it would scroll to the right the page numbers and keeps the
+    current one in the middle of the pager component. When in the last pages,
+    the pages won't scroll and just keep moving until the last page. Pager also contains
+    <first, previous, ..., next, last> pages.
+    This component takes into account custom parameters such as search and showPaused,
+    which could be added to the pages link in order to maintain the state between
+    client and server. It also allows to make a bookmark on a specific paging state.
+    :param current_page:
+        the current page number, 0-indexed
+    :param num_of_pages:
+        the total number of pages
+    :param search:
+        the search query string, if any
+    :param showPaused:
+        false if paused dags will be hidden, otherwise true to show them
+    :param window:
+        the number of pages to be shown in the paging component (7 default)
+    :return:
+        the HTML string of the paging component
+    """
+
+    def get_params(**kwargs):
+        params = []
+        for k, v in kwargs.items():
+            if k == 'showPaused':
+                # True is default or None
+                if v or v is None:
+                    continue
+                if not v:
```

Comment:
I think this if is guaranteed to be true at this point, you can probably remove the if and continue above and just do if v == False.

Reply:
Right, I just left the `if` to make it explicit since this parameter is kind of confusing given it's defaults. I can remove the if.

Reply:
Can add a comment or something, It's a little confusing this way, not a huge deal though. LGTM. Let me know when you're ready for me to merge.